spinalcordtoolbox/types.py

```diff
@@ -564,15 +590,25 @@ def compute_vertebral_distribution(self, disks_levels, label_reference='C1'):
 
                     if current_label in ['PMJ', 'PMG']:
                         if next_label in self.index_disk:
-                            self.dist_points_rel[i] = - (self.dist_points[i] - self.dist_points[self.index_disk[next_label]]) / abs(self.dist_points[self.index_disk[next_label]] - self.dist_points[self.index_disk[current_label]])
+                            self.dist_points_rel[i] = -(
+                                self.dist_points[i] - self.dist_points[self.index_disk[next_label]]
+                            ) / abs(
+                                self.dist_points[self.index_disk[next_label]] - self.dist_points[self.index_disk[current_label]]
+                            )
                         else:
-                            self.dist_points_rel[i] = (self.average_vert_length[current_label] - self.dist_points[i] + self.dist_points[self.index_disk[current_label]]) / self.average_vert_length[current_label]
+                            self.dist_points_rel[i] = (
+                                self.average_vert_length[current_label] - self.dist_points[i] + self.dist_points[self.index_disk[current_label]]
+                            ) / self.average_vert_length[current_label]
                     else:
                         next_label = self.regions_labels[str(self.list_labels[self.list_labels.index(self.labels_regions[self.l_points[i]]) + 1])]
                         if next_label in self.index_disk:
-                            self.dist_points_rel[i] = (self.dist_points[i] - self.dist_points[self.index_disk[current_label]]) / abs(self.dist_points[self.index_disk[next_label]] - self.dist_points[self.index_disk[current_label]])
+                            self.dist_points_rel[i] = (
+                                self.dist_points[i] - self.dist_points[self.index_disk[current_label]]
+                            ) / abs(self.dist_points[self.index_disk[next_label]] - self.dist_points[self.index_disk[current_label]])
                         else:
-                            self.dist_points_rel[i] = (self.dist_points[i] - self.dist_points[self.index_disk[current_label]]) / self.average_vert_length[current_label]
+                            self.dist_points_rel[i] = (
+                                self.dist_points[i] - self.dist_points[self.index_disk[current_label]]
+                            ) / self.average_vert_length[current_label]
```

Comment:
I'm going to say this is out of scope for this PR.

Part of the problem with fixing lints, is that they tend to be co-located with other kinds of dodgy. And if I'm going to refactor these few lines, I'm going to want to similarly fix the previous 50 lines, because they also have repeated values with very long names. But I don't want the git-blame for those changes to point back to this PR, because that's not what it's about.